frontend/src/components/common/AddAdminModal.tsx

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from "react";
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  Modal,
+  ModalHeader,
+  ModalCloseButton,
+  ModalContent,
+  ModalBody,
+  ModalFooter,
+  ModalOverlay,
+  Radio,
+  RadioGroup,
+  HStack,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { PlusOutlineIcon } from "./icons";
+
+const AddAdminModal = (): React.ReactElement => {
+  const { onOpen, onClose, isOpen } = useDisclosure();
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [confirmEmail, setConfirmEmail] = useState("");
+  const [hasJumpMathEmail, setHasJumpMathEmail] = useState<boolean | null>(
+    null,
+  );
+
+  const onModalClose = () => {
+    setHasJumpMathEmail(null);
+    onClose();
+  };
+
+  const onAddAdminClick = () => {
+    if (email !== confirmEmail) {
+      console.log("emails don't match");
+      return;
+    }
+    console.log("creating admin with data");
+    console.log(
+      `first name: ${firstName}\nlast name: ${lastName}\n email: ${email}`,
+    );
+  };
+
+  return (
+    <>
+      <Button
+        rightIcon={<PlusOutlineIcon />}
+        variant="primary"
+        my={2}
+        onClick={onOpen}
+      >
+        Add Admin
+      </Button>
+      <Modal isOpen={isOpen} onClose={onModalClose} size="6xl" isCentered>
+        <ModalOverlay />
+        <ModalContent p={2}>
+          <ModalHeader>
+            <Text textStyle="subtitle1" color="grey.400">
+              Add Admin
+            </Text>
+          </ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <FormControl isRequired>
+              <FormLabel color="blue.300" fontSize="20px" lineHeight="26px">
```

Comment:
Tried using `textStyle` here but it wouldn't apply, so these are hard-coded. This seems to be a known issue, according to this issue in the Chakra UI repo: https://github.com/chakra-ui/chakra-ui/issues/4566